docs-internal/api/nu-designer-openapi.yaml

```diff
@@ -3350,6 +3402,68 @@ components:
       type: object
     SqlParameterEditor:
       type: object
+    StatisticDto:
+      required:
+      - name
+      - value
+      type: object
+      properties:
```

Comment:
We don't plan sessions so the distinction between users is needles.

Reply:
> We don't plan sessions so the distinction between users is needles.

I'm not sure if I get it. But we still have multiple users right? So if we will be able to recognize some patterns in case of specific anonymous user, it can be somehow useful, and in the future if the pattern changed, we can ask the question to yourself, what happened.

Reply:
We discussed it internally, and since there won't be a session context, there is no sense in providing this kind of information. We will just collect a count of specific events.